### ISSUE TITLE
Failed to disable the default metrics

### ISSUE THREAD

**Issue by nghialv** [siimon/prom-client#44]
Hi,

```
var client = require('prom-client');
clearInterval(client.defaultMetrics());
client.register.clear();
```
When disabling the default metrics by this code. I got the following error:
```
Error: A metric with the name process_cpu_seconds_total has already been registered.
```

Using version is `5.0.0`

**Comment by SimenB** [comment]
Might have to

```js
var client = require('prom-client');
client.register.clear();
clearInterval(client.defaultMetrics());
client.register.clear();
```

That sucks, though...

**Comment by siimon** [comment]
Fixed in 5.0.1

**Comment by nghialv** [comment]
Thanks for your effort.
